src/components/data/listing/Listing.js

```diff
@@ -166,44 +169,48 @@ function Listing(props) {
             page,
             uploadsCompleted,
         ],
-        queryFn: getPagedListing,
-        config: {
-            enabled: !!path,
-            onSuccess: (respData) => {
-                trackIntercomEvent(IntercomEvents.VIEWED_FOLDER, {
-                    path,
-                });
-                setData({
-                    total: respData?.total,
-                    permission: respData?.permission,
-                    listing: [
-                        ...respData?.folders.map((f) => ({
-                            ...f,
-                            type: ResourceTypes.FOLDER,
-                        })),
-                        ...respData?.files.map((f) => ({
-                            ...f,
-                            type: ResourceTypes.FILE,
-                        })),
-                    ].map((i) => camelcaseit(i)), // camelcase the fields for each object, for consistency.
-                });
-            },
+        queryFn: () =>
+            getPagedListing(
+                path,
+                rowsPerPage,
+                orderBy,
+                order,
+                page,
+                uploadsCompleted
+            ),
+        enabled: !!path,
+        onSuccess: (respData) => {
+            trackIntercomEvent(IntercomEvents.VIEWED_FOLDER, {
+                path,
+            });
+            setData({
+                total: respData?.total,
+                permission: respData?.permission,
+                listing: [
+                    ...respData?.folders.map((f) => ({
+                        ...f,
+                        type: ResourceTypes.FOLDER,
+                    })),
+                    ...respData?.files.map((f) => ({
+                        ...f,
+                        type: ResourceTypes.FILE,
+                    })),
+                ].map((i) => camelcaseit(i)), // camelcase the fields for each object, for consistency.
+            });
         },
     });
 
     const { defaultsMappingError, isFetchingDefaultsMapping } = useQuery({
         queryKey: [DEFAULTS_MAPPING_QUERY_KEY],
         queryFn: getDefaultsMapping,
-        config: {
-            enabled: true,
-            onSuccess: (respData) => {
-                setInstantLaunchDefaultsMapping(respData?.mapping || {});
-            },
+        enabled: true,
+        onSuccess: (respData) => {
+            setInstantLaunchDefaultsMapping(respData?.mapping || {});
         },
     });
 
     const refreshListing = () =>
-        queryCache.invalidateQueries(
+        queryClient.invalidateQueries(
```

Comment:
Not sure if it's just in my local dev, but the `Refresh` button in the data listing no longer appears to have an effect.

Reply:
I could reproduce only by following these steps. It works on other flows.

- Go to DE. (Keep RQ dev panel open.)
- Navigate to `Data` without logging in. Data listing query is loaded in dev panel.
- Hit `Refresh`. This should fresh the view normally.
- Now login to the DE.
- After signing in with username and password, you will be redirected to same `Data` view. No Data listing query can be found in dev panel. 😕 
- Now hit `Refresh` again. This won't work bcos there is no query refresh 😩 .
- Now change from `Community Data` to your home folder. Still no Data listing query can be found in the dev panel .🤦🏽‍♂️ . `Refresh` won't work here s 

Am not really sure how to fix this yet.

Reply:
Well. I managed to reproduce this error in other ways too but then just when I was about to give up, i `RTFM` and found the solution 😌 
`refetch: (options: { throwOnError: boolean, cancelRefetch: boolean }) => Promise<UseQueryResult>
A function to manually refetch the query.`